src/firefly/js/charts/ui/Histogram.jsx

```diff
@@ -459,17 +458,20 @@ export class Histogram extends React.Component {
 
 Histogram.defaultProps = {
     desc: 'Sample Distribution',
-    binColor: '#d1d1d1'
+    binColor: '#d1d1d1',
+    xAxis: {},
+    yAxis: {}
 };
 
 
 // when more than one histogram is defined use series
 Histogram.propTypes = {
     series: PropTypes.arrayOf(PropTypes.object), // array of objects with data, binColor, and name properties
+    xAxis: PropTypes.object,
+    yAxis: PropTypes.object,
```

Comment:
It may be useful if you made this a shape type with defined props.  just a thought.  I am not sure how useful it is.